content/post/rds-automating-designer-to-developer-handoff/index.md

```diff
@@ -0,0 +1,143 @@
+---
+title: "RDS: Automating Code and Asset Generation From Figma"
+date: 2021-10-20
+excerpt: "Designer-to-developer handoff is an important part of the product development life cycle. It takes place when the design deliverables are ready to be provided to developers for implementation."
+authors:
+  - Gianne Chiong
+hero: /post/rds-automating-designer-to-developer-handoff/images/rds_logo.png
+draft: true
+license: cc-by
+seo:
+  canonical: https://commerce-engineer.rakuten.careers/entry/en/tech/0045
+---
+
+Hi, this is Gian from the E-Commerce Mobile Architect Group. The mission of our group is to improve productivity by creating tools to automate tasks and providing technical support for the e-commerce mobile app development teams. In this article, we will discuss about one of the developer tools we created to automate designer-to-developer handoff.
+
+# Overview
+
+Designer-to-developer handoff is an important part of the product development life cycle. It takes place when the design deliverables are ready to be provided to developers for implementation.
+
+In the E-Commerce Mobile App Development Department, the design deliverables are mainly created using [Figma](https://www.figma.com). The design in Figma is considered as the single-source-of-truth from which the app development team derives the product's UI. As such, it is critical to ensure that it is formatted in a way that can be clearly understood by the developers to avoid any inconsistencies between the design and the final product.
+
+# Design Deliverable Prerequisites
```

Comment:
All the headers are level 1 (`#`), which is normally reserved for the document's title. Could you change them to level 2 (`##`) ?

Reply:
No prob. Updated.